orchestrator-salt/src/main/resources/salt/salt/postgresql/upgrade/embedded.sls

```diff
@@ -0,0 +1,116 @@
+{% set original_postgres_version = salt['pillar.get']('postgres:upgrade:original_postgres_version', '10') %}
+{% set original_postgres_directory = salt['pillar.get']('postgres:upgrade:original_postgres_directory', '/dbfs/pgsql') %}
+{% set original_postgres_binaries = salt['pillar.get']('postgres:upgrade:original_postgres_binaries', '/dbfs/tmp/pgsql-10') %}
+
+{% set new_postgres_version = salt['pillar.get']('postgres:upgrade:new_postgres_version', '11') %}
+{% set new_postgres_directory = salt['pillar.get']('postgres:upgrade:new_postgres_directory', '/dbfs/pgsql-' + new_postgres_version) %}
+{% set new_postgres_log_directory = salt['pillar.get']('postgres:upgrade:new_postgres_log_directory', '/dbfs/pgsql-' + new_postgres_version + '/log') %}
+{% set new_postgres_binaries = salt['pillar.get']('postgres:upgrade:new_postgres_binaries', '/usr/pgsql-11') %}
+
+{% if salt['file.file_exists']('/usr/pgsql-' + new_postgres_version + '/bin/psql') and salt['file.directory_exists'](original_postgres_binaries) %}
+
+stop-postgresql:
+  service.dead:
+    - name: postgresql
+    - enable: False
+
+{% set command = 'systemctl show -p FragmentPath postgresql-' + new_postgres_version %}
+{% set unitFile = salt['cmd.run'](command) | replace("FragmentPath=","") %}
+
+{{ new_postgres_directory }}:
+  file.directory:
+    - user: postgres
+    - group: postgres
+    - mode: 700
+
+{{ new_postgres_log_directory }}:
+  file.directory:
+    - user: root
+    - group: root
+    - mode: 755
+
+init-db-with-utf8:
+  cmd.run:
+    - name: rm -rf {{ new_postgres_directory }}/data && runuser -l postgres -s /bin/bash sh -c '{{ new_postgres_binaries }}/bin/initdb --locale=en_US.UTF-8 {{ new_postgres_directory }}/data > {{ new_postgres_directory }}/initdb.log'
+    - unless: grep -q UTF-8 {{ new_postgres_directory }}/initdb.log
```

Comment:
`test -f {{ postgres_directory }}/data/PG_VERSION` --> this is not needed anymore?